### DISCUSSION TITLE
[3-1]

### DISCUSSION THREAD

**Discussion by moonuckjoo** [bbang-jun/2023-OOP-Mentoring#18] [General]
if (curNode == head) { // 1. head를 삭제하는 경우
		if (head->getNext() == NULL) { // list에 head만 존재하는 경우
			delNode = head;
			delete head;
			// head = NULL; 팁1
올려주신 링크드리스트 예제입니다. 여기에서  delNode = head라는 문장이 왜 필요한지 모르겠습니다. ㅠ 어차피 리스트에 헤드만 있고 헤드를 지우고 재생성할건데 굳이 필요한건가요?

**Comment by bbang-jun** [comment]
delete head;
head = null;
만 해주어도 됩니다!
head 삭제 시 delNode 변수로 같이 설명드린 이유는 중간 삭제, tail 삭제의 경우랑 같이 설명드릴 때 통일성 있게 설명해서 쉽게 이해시켜 드리기 위해서입니다.
